.travis.yml

```diff
@@ -7,5 +7,5 @@ jdk:
 before_install:
   - export PATH=${PATH}:./vendor/bundle
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test:scalafmtCheck scalafmtSbtCheck "project core" test:run "project scalacheck" test:run
+  - sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test:scalafmtCheck scalafmtSbtCheck "project core" test:run "project scalacheck" test:run "project playJson" test:run
```

Comment:
I think I should have thought about it earlier, (I mean, we *do* have a `root` module aggregating all other modules already).
Anyway, this will be indirectly fixed when I update this PR to accommodate the changes introduced by #39.